lms/djangoapps/support/static/support/js/collections/certificate.js

```diff
@@ -7,14 +7,23 @@
 
                 initialize: function(options) {
                     this.userQuery = options.userQuery || '';
+                    this.courseID = options.courseID || '';
                 },
 
                 setUserQuery: function(userQuery) {
                     this.userQuery = userQuery;
                 },
 
+                setCourseQuery: function(courseQuery) {
+                    this.courseID = courseQuery;
```

Comment:
I don't really like this "Query" terminology we're using -- "Filter" makes more sense to me.  Is there a specific reason that we've decided on Query?  ie, it's a Backbone convention or something?